lightning_pose/utils/predictions.py

```diff
@@ -813,34 +810,34 @@ def seconds_to_hms(seconds):
 
     # add marker to each frame t, where t is in sec
     def add_marker_and_timestamps(get_frame, t):
```

Comment:
Minor code health things.
The only behavior change is, 

instead of printing "Skipped frame" once for each marker, it's once for the whole frame.